src/commands/githubActions/endToEndTest/testCheckers.ts

```diff
@@ -43,10 +47,17 @@ export async function executeTestCheckers({
         console.log(chalk.green(`  ✓ Healthcheck endpoint returned 200`))
       )
       .catch(e => errors.push(e));
-  if (network)
+
+  /** 
+    * The attestation check will only be performed when a few requeriments are met:
+    * - The package to test is a staker package or the default execution package (execution client needs to be on sync in order to attest)
+    * - The test is the "testPackageInstallAndUpdate" and not the "testPackageInstallFromScratch" (the check can take up to 20 min to be completed).
+  **/
+  if (network && isUpdateTest && getIsStakerPkg(dnpName) && dnpName !== getStakerConfigByNetwork(network).executionClient.dnpName && !process.env.TEST) {
```

Comment:
This if check is too complex for human redability, either try to simplify it or create a utility function `getExecuteProofOfAttestation`

Notice that in order to let this funcion as much pure as possible, there should be just one var to execute or not the proof of attestation, ideally calculated before and passed as an argument